lib/rdf.js

```diff
@@ -127,6 +127,28 @@ function createRDF(manifest) {
     });
   }
 
+  if (manifest.locales) {
+    for (var locale in manifest.locales){
+      var l10n = manifest.locales[locale];
+      var l10nDescription = {
+        "em:locale": locale,
+        "em:name": l10n.title || manifest.title || manifest.name || "Untitled",
+        "em:description": l10n.description || manifest.description || undefined,
```

Comment:
yeah, no idea what jsontoxml does for other falsy values, so it's probably best to leave it